test/lib/descriptor.spec.js

```diff
@@ -275,6 +291,11 @@ describe('FileDescriptor', function() {
       const fd = new FileDescriptor(flags('ax+'));
       assert.isTrue(fd.isRead());
     });
+
+    it('opened for reading (O_CREAT)', function() {
+      const fd = new FileDescriptor(constants.O_CREAT);
+      assert.isTrue(fd.isRead());
```

Comment:
I don't think we want to treat `O_CREATE` as a read mode.

Reply:
The description of this test might be misleading. It should be `O_CREAT | O_RDONLY` which equals `O_CREAT`.

Reply:
Tests are updated.

Reply:
@tschaub can you review again?